train_gpt2.cu

```diff
@@ -601,12 +601,12 @@ __global__ void matmul_backward_bias_kernel_faster(float* dbias, const float* do
     }
     // write the final result (at thread 0) to global memory
     if (tid == 0) {
-        dbias[o] = shared[0];
+        dbias[o] += shared[0];
```

Comment:
every change is += -> =, except here = -> += ?

Reply:
this is the bias kernel, right. So if we're doing gradient accumulation over multiple steps, we would want to add these up, right?
Technically, this isn't part of the memory changes, but when checking each kernel, I noticed this.

Reply:
But aren't we able to keep it as = like all the other kernels?